specutils/tests/test_loaders.py

```diff
@@ -875,7 +1026,7 @@ def test_wcs1d_fits_compressed(compress, tmp_path):
     # Try again without compression suffix:
     with warnings.catch_warnings():
         warnings.simplefilter('ignore', FITSFixedWarning)
-        os.system(f'mv {tmpfile}{ext[compress]} {tmpfile}')
+        os.system(f'mv {tmpfile}.{ext[compress]} {tmpfile}')
```

Comment:
Side note: Why call `os.system` instead of `shutil`? Anyways, I guess fixing this is out of scope.

Reply:
I don't see how "mv" cares about compression? https://docs.python.org/3/library/shutil.html#shutil.move

Reply:
Not at all, I guess, but it's kind of the direct counterpart to `os.system(f'{compress} {tmpfile}')` just above.